storage_service/locations/models/s3.py

```diff
@@ -100,10 +100,16 @@ def _ensure_bucket_exists(self):
             if error_code != "NoSuchBucket":
                 raise StorageException(err)
             LOGGER.info("Creating S3 bucket '%s'", self._bucket_name())
-            self.client.create_bucket(
-                Bucket=self._bucket_name(),
-                CreateBucketConfiguration={"LocationConstraint": self.region},
-            )
+            # LocationConstraint cannot be specified if it us-east-1 because it is the default, see: https://github.com/boto/boto3/issues/125
+            if self.region == 'us-east-1':
+                self.client.create_bucket(
+                    Bucket=self._bucket_name()
```

Comment:
This looks good, can you run `black` against the changed files, I think it will convert this line to something like:
```
+            if self.region == "us-east-1":
+                self.client.create_bucket(Bucket=self._bucket_name())
```
And then I think the build will pass okay! 

Black is found here: https://github.com/psf/black

Reply:
I totally messed up the rebasing process. I'm just going to resubmit the pull request.